cms/appresolver.py

```diff
@@ -218,6 +223,11 @@ def _get_app_patterns():
         current_site = None
     included = []
 
+    # If APP_RESOLVERS is loaded,
+    if APP_RESOLVERS.get(current_site, []):
+        return copy(APP_RESOLVERS.get(current_site, []))
```

Comment:
I would prefer to have the copy in the urls.py then, this way is more clear why is needed and logic falls on the caller instead of this function.